Testing/ContinuousIntegration/AzurePipelinesMacOSPython.yml

```diff
@@ -44,12 +44,14 @@ jobs:
 
     - bash: |
         python_executable=`which python3`
+        python_root_dir=$(dirname $(dirname ${python_executable}))
         cat > dashboard.cmake << EOF
         set(CTEST_BUILD_CONFIGURATION "Release")
         set(CTEST_CMAKE_GENERATOR "Ninja")
         set(BUILD_NAME_SUFFIX "-Python")
         set(dashboard_cache "
-          PYTHON_EXECUTABLE:FILEPATH=${python_executable}
+          Python3_ROOT_DIR:PATH=${python_root_dir}
```

Comment:
This change is not done for Windows. Intentional or oversight?

Reply:
@dzenanz The variable "PYTHON_EXECUTABLE" did not exist in the current windows path.  Perhaps it should, but I don't know the windows environment well enough to know what to set the variable to.  There is also no `which python3` shell script for windows either.

Can this intermediate refactoring be done separately from a windows/python refactoring?

Reply:
Sure! Just making sure it is not an oversight.